src/code/z_parameter.c

```diff
@@ -52,38 +293,753 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80112AFC.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80112B40.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/Interface_LoadItemIconImpl.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80112BE4.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/Interface_LoadItemIcon.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80112C0C.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/Item_Give.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_801143CC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80114978.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_801149A0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80114A9C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80114B84.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80114CA0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/Interface_HasEmptyBottle.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/Interface_HasItemInBottle.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80114FD0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80115130.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_801152B8.s")
+s16 sAmmoRefillCounts[] = { 5, 10, 20, 30 }; // Sticks, nuts, bombs
+s16 sArrowRefillCounts[] = { 10, 30, 40, 50 };
+s16 sBombchuRefillCounts[] = { 20, 10, 1, 5 };
+s16 sRupeeRefillCounts[] = { 1, 5, 10, 20, 50, 100, 200 };
+u8 Item_Give(PlayState* play, u8 item) {
```

Comment:
Leave a space between the data and the function please, makes it easier to read (and makes space for the documentation comment this is bound to need).